packages/babel-plugin-transform-class-properties/test/fixtures/source-maps/private-get/source-map-visual.txt

```diff
@@ -139,41 +145,44 @@
  (11:8-9) this.#met               <--  (11:48-50) this, _met
               ^                                       ^^
 
-(11:9-16) his.#method;            <--  (11:50-57) is, _method);
-              ^^^^^^^                                 ^^^^^^^
+(11:9-15) his.#method;            <--  (11:50-57) is, _method);
+              ^^^^^^                                  ^^^^^^^
 
-(11:16-17) thod;                  <--  (11:57-59) thod);
-               ^                                      ^^
+(11:15-16) ethod;                 <--  (11:57-58) thod);
+               ^                                      ^
+
+(11:16-17) thod;                  <--  (11:58-59) hod);
+               ^                                      ^
 
 (13:4-10)     inExpr(thi          <--  (12:4-10)     inExpr(bab
               ^^^^^^                                 ^^^^^^
 
-(13:10-16) Expr(this.#fie         <--  (12:10-11) Expr(babe
-               ^^^^^^                                 ^
+(13:10-11) Expr(this              <--  (12:10-11) Expr(babe
+               ^                                      ^
 
-(13:10-16) Expr(this.#fie         <--  (12:11-23) xpr(babelHelpers.cla
-               ^^^^^^                                 ^^^^^^^^^^^^
+(13:11-16) xpr(this.#fie          <--  (12:11-23) xpr(babelHelpers.cla
+               ^^^^^                                  ^^^^^^^^^^^^
 
-(13:10-16) Expr(this.#fie         <--  (12:23-24) pers.clas
-               ^^^^^^                                 ^
+(13:11-16) xpr(this.#fie          <--  (12:23-24) pers.clas
+               ^^^^^                                  ^
 
-(13:10-16) Expr(this.#fie         <--  (12:24-45) ers.classPrivateFieldGet2(_fi
-               ^^^^^^                                 ^^^^^^^^^^^^^^^^^^^^^
+(13:11-16) xpr(this.#fie          <--  (12:24-45) ers.classPrivateFieldGet2(_fi
+               ^^^^^                                  ^^^^^^^^^^^^^^^^^^^^^
 
-(13:10-16) Expr(this.#fie         <--  (12:45-46) Get2(_fie
-               ^^^^^^                                 ^
+(13:11-16) xpr(this.#fie          <--  (12:45-46) Get2(_fie
```

Comment:
This is another fix. Previously, `babelHelpers.classPrivateFieldGet2...` had no loc info so it was merged with the previous token (in this example, the `(` of `inExpr(`).

Now it's a bit weird that the closing paren of the `babelHelpers.classPrivateFieldGet2` call is mapped to the last character of the private name, but in practice it's ok since no error or breakpoint will ever point to it.